documentation/UserGuide.md

```diff
@@ -65,6 +65,12 @@ If you do not provide a mapconfig.xml file, a default list of ArcGIS Online base
 
 After the first launch, the app uses the bsaemap layers that it loaded previously. If you want to reset and re-read mapconfig.xml, you can [reset the map](#reset-the-map). Alternatively, you can manually go to the Android application settings, choose Squad Leader, and choose Clear Data. Then run the app and it will read mapconfig.xml again.
 
+#### GPS simulation
+
+By default, Squad Leader uses the device's location capabilities to display the user's current location on the map. This is the appropriate setting for real-world deployments, so most likely you don't need to do anything to change location settings. Read on if you are interesting in using simulated GPS.
+
+Squad Leader can use a user-specified GPX file or a built-in GPX file of track points in Monterey, California. An IT professional who wants to deploy Squad Leader with a simulation based on a GPX file can create a file named simulation.gpx, create a directory called SquadLeader on the device's main storage (not an SD card), and put simulation.gpx in the SquadLeader directory. Then Squad Leader will use simulation.gpx to simulate the user's location until the user chooses a location setting in the Squad Leader UI. After a user has chosen a location setting, simulation.gpx will not be used again unless the Squad Leader app's data is cleared in Android settings.
```

Comment:
Just a future maintenance tip, if you don't include/mention specifics like the "Monterey, California" area of interest, this doc is still complete andl reads the same, but doesn't need to be updated if this changes (& it probably will change at some point again)